omnibus/config/software/python3.rb

```diff
@@ -64,25 +65,66 @@
     block do
       FileUtils.rm_f(Dir.glob("#{install_dir}/embedded/lib/python#{major}.#{minor}/distutils/command/wininst-*.exe"))
     end
-  end
+  else
+    dependency "vc_redist_14"
+
+    vcrt140_root = "#{Omnibus::Config.source_dir()}/vc_redist_140/expanded"
+    ###############################
+    # Setup openssl dependency... #
+    ###############################
+    mkdir "externals/openssl-bin-3.0.15/amd64/include"
```

Comment:
Any chance we can avoid hardcoding the version here?

Reply:
Good point, done!